packages/core/services/managers/marker-manager.ts

```diff
@@ -23,13 +23,15 @@ export class MarkerManager {
       return Promise.resolve();
     }
     return m.then((m: Marker) => {
-      return this._zone.run(() => {
-        m.setMap(null);
-        this._markers.delete(marker);
-      });
+      m.setMap(null);
+      this._markers.delete(marker);
     });
   }
 
+  deleteMarkers(markers: AgmMarker[]): Promise<void> {
+    return Promise.all(markers.map(marker => this.deleteMarker(marker))).then(() => {});
```

Comment:
is this needed? `.then(() => {}`

Reply:
This is to keep typescript happy and to actually return a "void" result, it is not required, but otherwise it will return the resolved array of promises (ie array of voids ;-))...